modules/installation-azure-permissions.adoc

```diff
@@ -10,6 +10,8 @@ Your Microsoft Azure account must have the following roles for the subscription
 that you use:
 
 * `User Access Administrator`
+* `Contributor`
```

Comment:
If the user already has the `Owner` role, I guess it does not need `Contributor` any more.

Reply:
ack